src/txn/txn_ckpt.c

```diff
@@ -732,7 +732,8 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     WT_TXN_GLOBAL *txn_global;
     WT_TXN_ISOLATION saved_isolation;
     wt_timestamp_t ckpt_tmp_ts;
-    uint64_t fsync_duration_usecs, generation, time_start, time_stop;
+    uint64_t fsync_duration_usecs, generation, time_start_fsync, time_stop_fsync, time_start_hs,
```

Comment:
This should be split across two different lines instead of wrapping.
```
uint64_t foo, bar, foobar, ...;
uint64_t radish, ...;
```

Reply:
oohhh will fix that

Reply:
thanks